packages/gatsby-plugin-sitemap/README.md

```diff
@@ -63,4 +66,5 @@ plugins: [
 ]
 ```
 
+If you are using `pathPrefix` in `gatsby-config.js`, you must run `gatsby build --prefix-paths` to build a sitemap with the prefix included. See [path prefix docs](https://www.gatsbyjs.org/docs/path-prefix/#production-build).
```

Comment:
Curious why was this added here? There's already a standalone doc page.

Reply:
It was something that tripped me up when testing, largely since I haven't used or needed `pathPrefix` before.

I need to take another shot at the `pathPrefix` stuff before this ready to merge.

Reply:
Ok, makes sense. Please remove it though as it's not relevant to this plugin other than the plugin should support it.